submissions/HannaSyn/frogger-game/app.js

```diff
@@ -0,0 +1,176 @@
+const cellHeight = 83;
+const cellWidth = 101;
+const totalRow = 6;
+const totalCol = 5;
+
+class Position {
+    minY;
+    minX;
+    maxY;
+    maxX;
+    initX;
+    initY;
+    x;
+    y;
+
+    constructor( row, col ) {
+        this.initX = (cellWidth * col) - (cellWidth);
+        this.initY = (cellHeight * row) - (cellHeight / 2);
+        this.x = this.initX;
+        this.y = this.initY;
+        this.maxY = cellHeight * (totalRow - 1);
```

Comment:
This value looks like a global maximum (not specific to a particular instance). Maybe is it not worth putting it into every item, but rather leave it as a global constant, what do you think?

Reply:
I think, that constants are relate to the Position class, because they means maximum and minimun positions of objects on the field.